django_mongodb/client.py

```diff
@@ -4,11 +4,44 @@
 
 
 class DatabaseClient(BaseDatabaseClient):
-    executable_name = "mongo"
+    executable_name = "mongosh"
 
     @classmethod
     def settings_to_cmd_args_env(cls, settings_dict, parameters):
-        raise NotImplementedError
+        options = settings_dict.get("OPTIONS", {})
+        uri = options.get("uri")
+        args = [cls.executable_name]
+        if not uri:
+            host = settings_dict["HOST"]
+            port = settings_dict.get("PORT")
```

Comment:
There are standard Django settings that will always be populated in `settings_dict`, even if not specified in the user's DATABASES, so you don't need to use `.get()` for them.